<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialsReportForm</class>
 <widget class="QWidget" name="MaterialsReportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчет по материалам</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="materialBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="workBox">
       <item>
        <property name="text">
         <string>ТР</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>КР</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="onlyDoneBox">
       <property name="text">
        <string>Только выполненные</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="materialTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Количество</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Работа</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
